apps/api/src/routers/demo.py

```diff
@@ -15,3 +17,18 @@ async def me(username: Annotated[Union[str, None], Cookie()] = None) -> str:
 async def square(value: Annotated[int, Form()]) -> int:
     """Calculate the square of the value."""
     return value * value
+
+
+@router.post("/user")
```

Comment:
Thought: since this is just fetching data, I think it would be more appropriate to specify the endpoint as a GET method?